main/admin/course_add.php

```diff
@@ -42,7 +42,14 @@
 $form->addElement('header', $tool_name);
 
 // Title
-$form->addText('title', get_lang('Title'), true);
+$form->addText(
+    'title',
+    get_lang('Title'),
+    true,
+    array(
+        'id' => 'title'
```

Comment:
It seems dangerous to me to add an HTML ID on a page (any page) as "title". This is just a very common word and just a small lack of awareness could make you create another element with the same ID. Please change to "course-title" or something like that.